README.md

```diff
@@ -800,7 +800,7 @@ to quickly experiment with themes and other customizations.
 | `--playground-tab-bar-foreground-color`            | ![](images/colors/000000.png) `#000000`                                                     | Text `color` of inactive file-picker tabs                                                                                          |
 | `--playground-tab-bar-active-color`                | ![](images/colors/6200EE.png) `var(--playground-highlight-color, #6200EE)`                  | Text `color` of active file-picker tab                                                                                             |
 | `--playground-tab-bar-indicator-color`             | ![](images/colors/6200EE.png) `var(--playground-highlight-color, #6200EE)`                  | `color` of active file-picker tab indicator (use `transparent` to hide)                                                            |
-| `--playground-tab-bar-font-size`                   | `0.85em`                                                                                    | `font-size` of tab titles in the file-picker tab bar                                                                               |
+| `--playground-tab-bar-font-size`                   | `14px`                                                                                      | `font-size` of tab titles in the file-picker tab bar                                                                               |
```

Comment:
Nit: Why the change of font-size default?

Reply:
I thought it was weird that we used `em` for this value, but `px` for the code font size, and most of the other properties. I think that default was inherited from `mwc-tab`, which we used to use, before we switched to our own tab implementation. Updated the CHANGELOG to mention this.